src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseAdmCloseService.java

```diff
@@ -21,8 +21,8 @@
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.ADMIN;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.CLOSED_STATE;
 import static uk.gov.hmcts.ethos.replacement.docmosis.constants.TableMarkupConstants.ADDITIONAL_INFORMATION;
+import static uk.gov.hmcts.ethos.replacement.docmosis.constants.TableMarkupConstants.APP_DETAILS_DETAILS;
```

Comment:
Just a small question, why do we call it APP_DETAILS_DETAILS?

Reply:
It is an existing constant with some history. Happy to rename it in this ticket.